src/Base/AMReX.cpp

```diff
@@ -45,4 +45,13 @@ void init_AMReX(py::module& m)
     m.def("finalize", py::overload_cast<AMReX*>(&Finalize));
 
     m.def("space_dim", [](){ return AMREX_SPACEDIM; });
+
+    m.def("have_mpi",
```

Comment:
actually that would make a great property
```suggestion
    m.def_property_readonly("have_mpi",
```

Reply:
@ax3l I don't think you can do `def_property_readonly` on a module object.

Reply:
true, sorry!